Deployment/Kubernetes/TensorRT-LLM_Autoscaling_and_Load_Balancing/chart/pvc/pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: efs-autoscaling-pv-2
```

Comment:
we should pick a more meaningful, and less AWS specific name for this.

Reply:
Moved all pvc charts for aws under pvc_aws folder